src/Contracts/MASA.Scheduler.Contracts.Server/Model/Job.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace MASA.Scheduler.Contracts.Server.Model;
+
+public class Job
+{
+    public DateTime CreationTime { get; set; }
+
+    public int Id { get; set; }
+
+    public string OrderNumber { get; set; } = "";
```

Comment:
OrderNumber?

Reply:
Already changed in branch feature/domainAggregates